src/constants.ts

```diff
@@ -9,6 +9,9 @@ export const REPOS = {
   },
 };
 
+export const ORBS = 'orbs';
+export const ORB_OWNER = 'electron';
```

Comment:
Feels like this is named a bit confusingly - it's the owner organization for the repos, right?